lib/quiethours.cc

```diff
@@ -16,15 +17,15 @@ Napi::Boolean Method(const Napi::CallbackInfo& info) {
 
   #ifdef _WIN32
   HKEY hKey;
-  LPTSTR lpValueName = "NOC_GLOBAL_SETTING_TOASTS_ENABLED";
+  std::string lpValueName = "NOC_GLOBAL_SETTING_TOASTS_ENABLED";
```

Comment:
It ought to be

```diff
- LPTSTR lpValueName = "NOC_GLOBAL_SETTING_TOASTS_ENABLED";
+ LPCTSTR lpValueName = TEXT("NOC_GLOBAL_SETTING_TOASTS_ENABLED");
```

but do not worry. [It's not gonna work even if you fix it](https://stackoverflow.com/a/54233600):

> Since [a recent major update to Windows 10](https://www.petri.com/focus-assist-replaces-quiet-hours-in-redstone-4), "Quiet Hours" has changed to "Focus Assist" - and it seems to work differently, so previous answers don't apply any more.

A replacement https://github.com/bitdisaster/windows-focus-assist is available, but:

> This native node module uses an unsupported/undocumented API to get the status of the Focus Assist.  …
>
> ⚠ The API used by windows-focus-assist can change/break at any time in the future. ⚠

Reply:
the point is to make this work for people who still use OS version that supports this

Reply:
Focus assist was introduced back in 2018. Multiple OS upgrades have been released since then. Not only that, Windows 10 will reach EoL the next year.